vercel.json

```diff
@@ -1,10 +1,14 @@
 {
-  "crons": [{
-    "path": "/api/sync",
-    "schedule": "0 2 * * *"
-  }],
-  "rewrites": [{
-    "source": "/assets/:path",
-    "destination": "/assets/:path"
-  }]
-}
\ No newline at end of file
+  "crons": [
+    {
+      "path": "/api/sync",
+      "schedule": "30 11 * * *"
```

Comment:
I would make a separate cron job that runs every hour. Or more specifically, every hour during the workday ([https://crontab.guru/#0_7-20_*_*_*](https://crontab.guru/#0_7-20_*_*_*)).

... functions are cheap, this would be less than 400 invocations per month and cost us nothing ;)

Reply:
I see, then thats ok. Maybe in the future we can use something like https://cron-job.org/de/ to improve notifications, but its fine for now.